websites/G/G2G/presence.ts

```diff
@@ -0,0 +1,66 @@
+const presence = new Presence({
+		clientId: "1073167802296438875",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "https://i.imgur.com/XkzTOsp.png",
+		startTimestamp: browsingTimestamp,
+	};
```

Comment:
let's first create these variables so we can use the through out the code
```suggestion
	},
	{pathname, href} = document.location,
	path = pathname.split("/");
```